src/operator/leaky_relu-inl.h

```diff
@@ -225,7 +242,11 @@ class LeakyReLUProp : public OperatorProperty {
     const TShape &dshape = in_shape->at(leakyrelu::kData);
     if (dshape.ndim() == 0) return false;
     if (param_.act_type == leakyrelu::kPReLU) {
-      in_shape->at(leakyrelu::kGamma) = TShape(Shape1(dshape[1]));
+      const TShape &gshape = in_shape->at(leakyrelu::kGamma);
+      if (gshape.Size() != 1)
```

Comment:
gshape could be empty, in which case Size is undefined.
Also if gshape.Size is 1, it could be (1,1), which is invalid

Reply:
So, I should check for both ndim and shape_[0] then. How do I check whether it’s undefined?

Reply:
if gshape is empty gshape.ndim() would be 0